### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by TadesseDev**
Changes introduced. 😎
---
> For each URL I have created 👋🏼
- A route.✔
- An action i the correct controller.✔
- A view file.✔
> Note: view files contain only place holder for the time being.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def post; end
+
+  def posts; end
```
Comment (youmari) [inline app/controllers/users_controller.rb]: - [x] you should create a separate controller for `Posts` as in the requirements and you should add `actions` like `index` and `show` in both controllers `users` and `posts`

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get 'users/', to: 'users#index'
+  get 'users/:user_id/posts/:id', to: 'users#post'
+  get 'users/:user_id/posts', to: 'users#posts'
+  get 'users/:id', to: 'users#user'
```
Comment (youmari) [inline config/routes.rb]: - [x] the way that creating routes it's more verbose you can use `resources` it is much cleaner and you can only specify the `index` and `show` action  like this: 
```RUBY
resources :users, only: %i[index show] do 
    resources :posts, only: %i[index show]
end

File: app/views/users/posts.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<h1> this is a multi posts view</h1>
```
Comment (youmari) [inline app/views/users/posts.html.erb]: - [x] Also you should create a separate folder `posts` in views and its index and show pages

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Tadesse-Alemayehu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ PRO PR
✔️ Git flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
thank you @youmari for the thorough review. I'll make sure to add those changes before the next review.

**Review by brytebee — APPROVED** [review]
Hi, @Tadesse-Alemayehu👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- App functions as it should.✔️
- Excellent communication with the previous reviewer.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



##

###  **Optional Suggestions:**
- N/A

##

Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
Thank you @brytebee, Nice to have your view on this. 🙏🏼

**Pull request merged**